project_description/models/account_analytic_account.py

```diff
@@ -10,6 +10,5 @@ class account_analytic_account(models.Model):
 
```

Comment:
[Here](https://github.com/adhoc-dev/project/blob/7a30b818804954783bc2495d82582e3a5f0773b5/project_description/models/account_analytic_account.py#L8) the name of the model should be using the Camecase convencion